electron/src/renderer/components/Context.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const Context = createContext({});
```

Comment:
does this really need to be its own file? Especially since this is just a generic createContext and only used in 1 place (App.jsx)

Reply:
used in a bunch of files, anywhere that the `useContext` hook is imported. 4 files as of now.

Reply:
oh i see makes sense, thanks for clarifying